rfcs/0002-typography-scaling.md

```diff
@@ -0,0 +1,162 @@
+  - [Custom Scaling](#custom-scaling)
+- [Adoption strategy](#adoption-strategy)
+- [How do we educate people?](#how-do-we-educate-people)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+Typography is an integral part of web design. The user who is consuming information from our dashboard should not face any challenges while reading/analyzing data. Therefore we need to make sure that the type element should scale well across multiple devices (desktops/mobile/tablets/etc.).
+
+# Basic Example
+This is how the typography will look after the implementation of type scales.
+
+#### **On Desktop**
+<img width="600" src="./images/website-example-desktop.png" />
+<img width="600" src="./images/dashboard-example-desktop.png" />
+
+---
+
+#### **On Mobile**
+<div style="display:flex; align-items: flex-start">
+<img width="300" src="./images/website-example-mobile.png" />
+<img width="300" src="./images/dashboard-example-mobile.png" />
+</div>
+
+<br>
+
+# Motivation
+### **Why do we need a type scale?**
+We need our users to access our products anywhere, anytime. To make sure they can do so, we need to be present on almost every device possible especially on portables (mobiles & tablets). Every device offers a different experience and we need to make sure that we are flexible in that case. One of the major aspects of that experience is **readability** and therefore we are focusing on **typography scaling** across multiple devices.
+
+### **Use Cases**
+There can be several use-cases when a user wants to access the dashboard on multiple devices. Few of those can be,
+- **On Mobiles** (user can check for settlement numbers while commuting/traveling in a metro/bus)
+- **On Tablets** (user can capture a payment while on-the-go to their home back from the office)
+- **On Desktops** (user can request for access to the international payment via the dashboard)
+
+Or in-general a user can visit the company's marketing/landing page and consume information about us and our offerings.
+
+### **Expected Outcome**
+A typography system that is more readable/consumable across different devices (mobiles/tablets/desktops).
+
+# Detailed Design
+We are following a [**Major Second**](https://type-scale.com/?size=14&scale=1.125&text=A%20Visual%20Type%20Scale&font=Lato&fontweight=400&bodyfont=Poppins&bodyfontweight=400&lineheight=1.65&backgroundcolor=white&fontcolor=%23333&preview=false) modular scale (1.125) for our typography system but and that currently has two major limitations,
+- **Does not cater for landing/marketing pages.** Which are generally much larger compared to the size-ratio on the dashboard (Maybe a [Major Third](https://type-scale.com/?size=14&scale=1.125&text=A%20Visual%20Type%20Scale&font=Lato&fontweight=400&bodyfont=Poppins&bodyfontweight=400&lineheight=1.65&backgroundcolor=white&fontcolor=%23333&preview=false) scale will work better). This is currently being looked by the comm. design team based on the same principles.
```

Comment:
These limitations shall go under `Drawbacks/Constraints` section

Reply:
Yes, I added it there only at first, but this doesn't make sense as the user will need this context before diving into the whole typography scaling thing.